test/EcdsaMultikey.spec.js

```diff
@@ -24,6 +24,20 @@ describe('EcdsaMultikey', () => {
     });
   });
 
+  describe('algorithm', () => {
+    it('createSigner should export proper algorithm', async () => {
+      const keyPair = await EcdsaMultikey.from(mockKey);
+      const signer = keyPair.signer();
+      signer.algorithm.should.equal(ECDSA_CURVE.P256);
```

Comment:
```suggestion
      signer.algorithm.should.equal('P-256');
```
This may be a minor nit. In the test we would want to make an assertion against a known string value instead of the variable found in the source code.

It may be a stylistic choice so I'll include Dave on this one.

cc: @dlongley

Reply:
Yes, it's a good idea to test against a specific string here.

Reply:
OK, I see that you merged the PR before I could make this change. Since there are a number of instances in the test where we are using imported variables, do you recommend that I dedicate a PR to replacing those with known values?

Reply:
@kezike Yes, that would be fine.

Reply:
Sorry about that -- yes, another PR just to update the tests is fine.